### PR TITLE
Add log when trad not found

### PR DESCRIPTION
**Pull request by SelrahcD**

### CODE REVIEW

**Comment by maffpool** [comment]
:+1:

File: src/Translation/Manager.php [hunk 1]
```diff
@@ -243,6 +264,10 @@ public function translate($key, $lang = null)
                 $this->addNotFound($key, $lang);
             }
 
+            if($key === $translated) {
+                $this->logger->warning('Translation not found', array('key' => $key, 'lang' => $lang));
```
Comment (jjsaunier) [inline src/Translation/Manager.php]: Info or notice, warning is not appropriated

Comment (maffpool) [inline src/Translation/Manager.php]: Don't agree with you @ProPheT777 

```
WARNING (300): Exceptional occurrences that are not errors. Examples: Use of deprecated APIs, poor use of an API, undesirable things that are not necessarily wrong.
```

Comment (maffpool) [inline src/Translation/Manager.php]:
```
NOTICE (250): Normal but significant events.
```

For me, a not found translation is not a "Normal" event

Comment (SelrahcD) [inline src/Translation/Manager.php]: Same logic as @maffpool when I was carefully crafting that piece of code. <3

Comment (jjsaunier) [inline src/Translation/Manager.php]: On my side, a translation can not exist in specific language and let fallback on default, so a missing translation key can be the right behavior. So if you don't support that, yeah it have sense to log as warning.

Comment (SelrahcD) [inline src/Translation/Manager.php]: Warning it is then :)

**Comment by SelrahcD** [comment]
Good to go ?
v2.1.0 ?

**Pull request merged**